Update NegativeUNL page (#2619)
XRPLF/xrpl-dev-portal: docs/concepts/consensus-protocol/negative-unl.md

## Background

> Each server in the XRP Ledger protocol independently decides a ledger version is validated when a quorum of its trusted validators agree. A server's trusted validators are defined by that server's UNL (Unique Node List), a list of validators it trusts not to collude. By default, servers are configured to trust any validator on at least one of two lists, one UNL published by the XRP Ledger Foundation and one UNL published by Ripple.

Reviewer: ```suggestion
Each server in the XRP Ledger protocol independently decides a ledger version is validated when a quorum of its trusted validators agree. A server's trusted validators are defined by that server's UNL (Unique Node List), a list of validators it trusts not to collude. By default, servers are configured to trust any validator on the UNL published by either the XRP Ledger Foundation or Ripple.
```

Author: I prefer to leave it as-is to emphasize that they're configured with two lists.

> Each server in the XRP Ledger protocol has its own UNL (Unique Node List), a list of validators it trusts not to collude, and each server independently decides when a ledger version is validated based on the consensus when enough of its trusted validators agree on a new ledger version. The standard quorum requirement is at least 80% of trusted validators must agree. By default, servers are configured to download two lists, one UNL published by the XRP Ledger Foundation and one UNL published by Ripple, and to use the union of those two lists as their UNL. (In other words, by default a server listens to the votes of any validator that is on at least one of those lists.)

Reviewer 2: Each server in the XRP Ledger protocol has its own UNL (Unique Node List), a list of validators it trusts not to collude. Each server independently decides when a ledger version is validated, based on the consensus when enough of its trusted validators agree on a new ledger version. 

The standard quorum requirement is at least 80% of trusted validators must agree. By default, servers are configured to download two lists: one UNL published by the XRP Ledger Foundation, and one UNL published by Ripple. Servers use the union of those two lists as their UNL. (In other words, by default, a server listens to the votes of any validator that is on at least one of those lists.)

Author: I ended up rewriting this a little more thoroughly:

> Each server in the XRP Ledger protocol independently decides a ledger version is validated when a quorum of its trusted validators agree. A server's trusted validators are defined by that server's UNL (Unique Node List), a list of validators it trusts not to collude. By default, servers are configured to trust any validator on at least one of two lists, one UNL published by the XRP Ledger Foundation and one UNL published by Ripple.
>
> The standard quorum requirement is 80% of trusted validators. If more than 20% of trusted validators go offline or become unable to communicate with the rest of the network, servers stop validating new ledgers because they cannot reach a quorum. This is a design choice to ensure that no transactions' outcomes can be changed after they are declared final. During such a situation, the remaining servers would still be online and could provide past and tentative transaction data, but could not confirm the final, immutable outcome of new transactions.

> The standard quorum requirement is 80% of trusted validators. If more than 20% of trusted validators go offline or become unable to communicate with the rest of the network, servers stop validating new ledgers because they cannot reach a quorum. This is a design choice to ensure that no transactions' outcomes can be changed after they are declared final. During such a situation, the remaining servers would still be online and could provide past and tentative transaction data, but could not confirm the final, immutable outcome of new transactions.

Reviewer: Would it be better to specify in the last line that no new ledgers can be validated?

Author: It already says in the first line that servers stop validating new ledgers, so I don't think it's necessary to repeat it.

> However, this means that the network could stop making forward progress if a few widely-trusted validators went offline. As of 2024-06-07, there are a total of 35 validators on the default UNLs, so the network would stop making forward progress if 8 or more of them were offline. Furthermore, if one or two validators are out for an extended period of time, the network has less room for disagreement between the remaining validators, which can make it take longer to achieve a consensus.

Reviewer 2: widely trusted [no hyphen]
